sdc/tests/tests_perf/__init__.py

```diff
@@ -1,2 +1,3 @@
 from sdc.tests.tests_perf.test_perf_unicode import *
 from sdc.tests.tests_perf.test_perf_series_str import *
+from sdc.tests.tests_perf.test_perf_series import *
```

Comment:
I am not happy with importing via *. It always make me confusing when I try to decide where a thing was imported from.

Reply:
@PokhodenkoSA do you propose to list all tests instead of asterisk?